spec/helpers/widget_helper_spec.rb

```diff
@@ -12,16 +12,19 @@
     end
 
     it 'should return "Awaiting classification" for "waiting_classification' do
-      expect(status_description(@info_request, 'waiting_classification')).to eq('Awaiting classification')
+      expect(status_description(@info_request, 'waiting_classification'))
+        .to eq('Awaiting classification')
     end
 
     it 'should call theme_display_status for a theme status' do
-      allow(@info_request).to receive(:theme_display_status).and_return("Special status")
-      expect(status_description(@info_request, 'special_status')).to eq('Special status')
+      allow(InfoRequest).to receive(:theme_display_status).and_return("Special status")
+      expect(status_description(@info_request, 'special_status'))
+        .to eq('Special status')
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.